raiden/network/transport/matrix/transport.py

```diff
@@ -550,24 +553,35 @@ def send_global(self, room: str, message: Message) -> None:
 
         assert self._global_rooms.get(room_name), f'Unknown global room: {room_name!r}'
 
-        def _send_global():
-            text = JSONSerializer.serialize(message)
-            room = self._global_rooms[room_name]
-            self.log.debug(
-                'Send global',
-                room_name=room_name,
-                room=room,
-                data=text.replace('\n', '\\n'),
-            )
-            room.send_text(text)
-
-        greenlet = self._spawn(_send_global)
-        greenlet.name = (
-            f'MatrixTransport.send_global '
-            f'node:{pex(self._raiden_service.address)} '
-            f'room:{room} '
-            f'user_id:{self._user_id}'
-        )
+        self._global_send_queue.put((room_name, message))
```

Comment:
You could use some kind of notifier mechanism for the send global worker/loop, as done with the `_RetryQueue._notify_event` (which is set to notify and reset on loop), so once you put something on the queue, you don't need to wait up to `retry_interval` for it to actually be sent.